### ISSUE TITLE
LBRY addon

### ISSUE THREAD

**Issue by stokito** [kpl-addons/plugin.video.odysee#3]
https://kodi.wiki/view/Add-on:LBRY

Sources
https://github.com/accumulator/plugin.video.lbry

Is it related to your plugin?

**Comment by heppix** [comment]
Nope. Odysee is targeted for odysee.com website. I dont know this lbry plugin :)

**Comment by stokito** [comment]
I just tried it and in the settings it shows odysee.com  api endpoint.
On the https://lbry.com/ site when click on Try LBRY it redirects to the odysee.com

**Comment by heppix** [comment]
Yeah. LBRY is a platform allowing share your content and earn bitcoins or something. Odysee is like a frontend for it from what I understand ^^ However this add-on is not related to our odysee. Maybe I should check later source and see some ideas to implement here. Unfortunately I'm busy currently back soon I will back to odysee and planning to introduce login and subscriptions view.

**Comment by karlo2105** [comment]
@heppen7 
You could add possibility to use local server too because public server don't play several videos unlike local one. Live videos are missing. Joining forces with @accumulator to improve LBRY would be better in my opinion.
